shell/platform/fuchsia/flutter/engine.cc

```diff
@@ -401,6 +412,7 @@ void Engine::Initialize(
            view_ref_focused = std::move(view_ref_focused),
            touch_source = std::move(touch_source),
            mouse_source = std::move(mouse_source),
+           registry = std::move(registry),
```

Comment:
nit: for readability, pointerinjector_registry here and elsewhere

Reply:
Done